psutil/tests/__main__.py

```diff
@@ -32,8 +32,6 @@
     TEST_DEPS.extend(["ipaddress", "unittest2", "argparse", "mock==1.0.1"])
 elif sys.version_info[:2] == (2, 7) or sys.version_info[:2] <= (3, 2):
```

Comment:
That `or X` looks like a bug, it means this statement is also true for Python 2.6.

Python 2.6 then has both `mock==1.0.1` and `mock` in `TEST_DEPS`.

See PR #1237.